azerothcore/azerothcore-wotlk
# fix(DB/creature): Remove Gossip from NPC Winkey (7770)

## Changes Proposed:
- Removes the gossip menu from Winkey in Tanaris. This was not added until cataclysm

## SOURCE:
[WoW-pedia](https://wowpedia.fandom.com/wiki/Winkey) 

> Winkey has been in the game since the beginning but had no dialogue prior to Cataclysm and no quests before Legion.

[Classic WoWhead](https://classic.wowhead.com/npc=7770/winkey#comments) 
>She doesn't START or END a quest..and doesn't appear to be in the MIDDLE of any quests.  If there is another npc that mentions her it could be implimented later in the game, but for NOW, she serves NO PURPOSE!


## Tests Performed:
- Checked in game.

## How to Test the Changes:
1. Teleport to Winkey with `.go creature id 7770`
2. Talk to Winkey
3. Apply Fix
4. Restart World Server
5. Should no longer be able to talk to Winkey.

## Known Issues and TODO List:
None.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
